Runtime/Conditions/PositionalConditions/ObjectInRangeCondition.cs

```diff
@@ -27,12 +29,40 @@ public class EntityData : IObjectInTargetData
             [DisplayName("Object")]
             public SceneObjectReference Target { get; set; }
 
+            [DataMember] 
+            [HideInTrainingInspector]
```

Comment:
I don't think these are required, at least they don't show in the json file:

![image](https://user-images.githubusercontent.com/6911992/115530244-e2915680-a293-11eb-98fe-c0a2cfe3b6fd.png)


```suggestion
```

Reply:
Also, not sure if intended to use this field as persistent but if it is not kept in the json then it might be lost and achieving its purpose.

Reply:
Let's see if it works without it :D

Reply:
😱